packages/rspack/tests/configCases/type/multiple-options/index.js

```diff
@@ -0,0 +1 @@
+it("should pass type checking", function () {});
```

Comment:
The type checking test is not running, you can remove it because rspack not test export type at now.

Reply:
Thank your for your review! This file is added deliberately as a placeholder. If it had been removed, the test would fail.
And btw, I added `test:tc` command to `package.json` to execute pure type checking for relevent config files (currently, as specified in `tsconfig.test.json`, only one file will be checked). So we are able to test the types exported by rspack now.